### PR TITLE
Release/1.0.0

### PR DESCRIPTION
**Pull request by cs-pranjali-munfan**

### CODE REVIEW

File: aws-feed/info.json [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+  "tags": [
+    "threatintel-inbound"
+  ],
+  "ingestion_supported": true,
+  "ingestion_modes": [
+    "scheduled"
+  ],
+  "ingestion_preferences": {
+    "modules": [
+      "threat_intel_feeds"
```
Comment (cs-pooja-singh) [inline aws-feed/info.json]: add indicators as well

File: aws-feed/operations.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+import requests
+from connectors.core.connector import get_logger, ConnectorError
+
+logger = get_logger('aws-feed')
+
+
+def check_health(config):
+    try:
+        return get_indicators(config, {}, True)
```
Comment (cs-pooja-singh) [inline aws-feed/operations.py]: can we make this a little lightweight by skipping parsing the response. we can just look at some status code and top level keys in the response

File: aws-feed/info.json [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+    {
+      "operation": "get_indicators",
+      "title": "Get Indicators",
+      "category": "investigation",
+      "annotation": "get_indicators",
+      "description": "Retrieve list of all indicators form AWS feed",
+      "parameters": [
+      ],
+      "enabled": true,
+      "output_schema": {
```
Comment (cs-pooja-singh) [inline aws-feed/info.json:140]: take the last pull time also as input and return only those feeds that were updated after that. look at office 365 feed for reference

File: aws-feed/operations.py (hunk 2 above)
Comment (cs-pranjali-munfan) [reply aws-feed/operations.py]: handled in get_indicators() actions.

**Pull request merged**